taxonomies/food/categories.txt

```diff
@@ -46540,6 +46540,9 @@ fr:Mozzarella de bufflonne
 it:Mozzarella di bufala
 nl:Buffelmozzarellas
 
+<en:Mozzarella 
+minimum_number_of_ingredients:en:3
```

Comment:
No !
1. never add a blank line this way
2. < mean (inherit from…) find the right entry, I already gave you the grep command…